src/main/webapp/app/core/course/shared/course-sidebar/course-sidebar.component.ts

```diff
@@ -64,24 +70,30 @@ export class CourseSidebarComponent implements OnChanges {
     isTestServer = input<boolean>(false);
     hasUnreadMessages = input<boolean>(false);
     communicationRouteLoaded = input<boolean>(false);
+    layoutService = inject(LayoutService);
 
     hiddenItems = signal<SidebarItem[]>([]);
     anyItemHidden = signal<boolean>(false);
 
     switchCourse = output<Course>();
     courseActionItemClick = output<CourseActionItem>();
     toggleCollapseState = output<void>();
+    activeBreakpoints: Signal<string[]>;
+    canExpand: Signal<boolean>;
```

Comment:
Why don't you directly asssign the computed signal here instead of in the constructor?

I think it is better to define it here, or am I missing some Angular guidelines?